README.md

```diff
@@ -218,6 +218,16 @@ Below is an example of the execution of a DFU procedure of the file generated ab
 nrfutil dfu serial -pkg app_dfu_package.zip -p COM3
 ```
 
+##### usb_serial
+Perform a full DFU procedure over a serial (UART) line. This command is the same as `serial` but initial ping is not performed. It provides the possibility of enabling the ping feature, which is always performed in case of `serial`.
```

Comment:
As a user, I would want to know when to use `usb_serial` instead of `serial`. I am assuming it has something to do with which type of device it is or how it is connected, but this is not mentioned here. Whether it pings or not seems like a transport-level implementation detail.

Reply:
Well, this one I would also like know, but I lack the hardware experience phrase it in detail. Maybe @IvanSanchez could contribute with a commit here to clarify.